cmd/peapod-to-fstree/main.go

```diff
@@ -46,7 +46,13 @@ func (s *onlyObjectsWithContainersStorage) Iterate(objHandler func(oid.Address,
 		}
 
 		return objHandler(addr, data, id)
-	}, nil)
+	}, func(addr oid.Address, err error) error {
+		_, found := s.containers[addr.Container()]
+		if !found {
```

Comment:
Still log the event, please.

Reply:
it will be the object we will stop at, if there is `return err` below, it will be pronted anyway

Reply:
it's another matter if you meant that you shouldn't stop...

Reply:
`errorHandler` that return non-nil `err` (current PR's state) stop iteration